src/deployment/deployment-hapi-plugin.ts

```diff
@@ -12,26 +12,21 @@ import DeploymentModule, {
   isRawDeploymentHostname,
 } from './deployment-module';
 
-import { gitlabHostInjectSymbol } from '../shared/gitlab-client';
 import { Logger, loggerInjectSymbol } from '../shared/logger';
 import memoizee = require('memoizee');
```

Comment:
Not related to this PR, but memoizee should probably by in the top import group. Is there also a reason it's not imported with `import * as memoizee from 'memoizee'`?